newsletter/validators.py

```diff
@@ -2,17 +2,20 @@
 from django.forms.utils import ValidationError
 from django.utils.translation import gettext_lazy as _
 
+from newsletter.settings import newsletter_settings
+
 
 def validate_email_nouser(email):
     """
     Check if the email address does not belong to an existing user.
     """
-    # Check whether we should be subscribed to as a user
-    User = get_user_model()
+    if newsletter_settings.VALIDATE_USER:
+        # Check whether we should be subscribed to as a user
+        User = get_user_model()
 
-    if User.objects.filter(email__exact=email).exists():
-        raise ValidationError(_(
-            "The e-mail address '%(email)s' belongs to a user with an "
-            "account on this site. Please log in as that user "
-            "and try again."
-        ) % {'email': email})
+        if User.objects.filter(email__exact=email).exists():
```

Comment:
Should it be email__iexact? Because "example@gmail.com" and "Example@gmail.com' would be the same user.